### PR TITLE
[CARBONDATA-632] Fix wrong comments of load data in CarbonDataRDDFactory.scala

### PR DESCRIPTION
**Pull request by chenliang613**
Fix wrong comments of load data in CarbonDataRDDFactory.scala:
1.Currently, not use NewHadoopRDD and DummyLoadRDD
2.Clean invalid code:
inputFormat match
{ case configurable: Configurable => configurable.setConf(hadoopConfiguration) case _ => }

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/588/

File: integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala [hunk 1]
```diff
@@ -559,11 +560,7 @@ object CarbonDataRDDFactory {
           CommonUtil.configSplitMaxSize(sqlContext.sparkContext, filePaths, hadoopConfiguration)
 
           val inputFormat = new org.apache.hadoop.mapreduce.lib.input.TextInputFormat
-          inputFormat match {
-            case configurable: Configurable =>
-              configurable.setConf(hadoopConfiguration)
-            case _ =>
-          }
+
```
Comment (jackylk) [inline integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala]: This empty line can be removed

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/598/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/600/

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**